docs/about/faq.md

```diff
@@ -44,6 +44,58 @@ For convenience, we also [build container images with headscale](../setup/instal
 we don't officially support deploying headscale using Docker**. On our [Discord server](https://discord.gg/c84AZQhmpx)
 we have a "docker-issues" channel where you can ask for Docker-specific help to the community.
 
+## Scaling / How many clients does Headscale support?
+
+It depends. As often stated, Headscale is not enterprise software and our focus
+is homelabbers and self-hosters. Of course, we do not prevent people from using
+it in a commercial/professional setting and often get questions about scaling.
+
+Please note that when Headscale is developed, performance is not part of the
+consideration as the main audience is considered to be users with a moddest
+amount of devices. We focus on correctness and chaising feature parity over
+time.
+
+To understand if you might be able to use Headscale for your usecase, I will
+explain two scenarios in an effort to explain what is the central bottleneck
+of Headscale:
+
+1. An environment with 1000 servers
+
+- they rarely change "move" (change their endpoints)
+- new nodes are added rarely
+
+2. An environment with 80 laptops/phones (end user devices)
+
+- nodes move often, e.g. switching from home to office
```

Comment:
```suggestion
1. An environment with 1000 servers

    - they rarely  "move" (change their endpoints)
    - new nodes are added rarely

2. An environment with 80 laptops/phones (end user devices)

    - nodes move often, e.g. switching from home to office
```

indentation is required otherwise its rendered as a mix of flat numbered/bulleted list.